app/models/concerns/indexable.rb

```diff
@@ -22,17 +21,12 @@ module Indexable
       end
 
       if instance_of?(DataciteDoi) || instance_of?(OtherDoi) || instance_of?(Doi)
-        Rails.logger.info "[Event Data Import Message] its a DOI #{aasm_state} #{to_jsonapi} "
         if aasm_state == "findable"
           changed_attributes = saved_changes
-          Rails.logger.info "[Event Data Import Message] changes -- #{aasm_state} -- #{changes}"
-          Rails.logger.info "[Event Data Import Message] saved_changes -- #{aasm_state} -- #{saved_changes}"
           relevant_changes = changed_attributes.keys & %w[related_identifiers creators funding_references aasm_state]
-          Rails.logger.info "[Event Data Import Message] before call -- #{aasm_state} -- #{relevant_changes} --- #{(created == updated)}"
-          Rails.logger.info "[Event Data Import Message] dates -- #{aasm_state} -- #{created} --- #{updated}  "
           if relevant_changes.any? || (created == updated)
             send_import_message(to_jsonapi)
-            Rails.logger.info "[Event Data Import Message] send to Event Data service. #{aasm_state} #{to_jsonapi} "
+            Rails.logger.info "[Event Data Import Message] State: #{aasm_state} Params: #{to_jsonapi} message sent to Event Data service."
```

Comment:
@richardhallett I feel we should have this log which confirms that we did send a message from Lupo to Sqs. Right now there is no way to verify if Lupo sent a message correctly to sqs or not.

I would love to hear your thoughts on this, as I am adding some logs which might increase storage cost.

Reply:
Something that might help if I understand this correctly. The AWS::SQS::Client.send_message does return a result -> https://docs.aws.amazon.com/sdk-for-ruby/v3/api/Aws/SQS/Client.html#send_message-instance_method.

So you could get the result (https://docs.aws.amazon.com/sdk-for-ruby/v3/api/Aws/SQS/Types/SendMessageResult.html) by simply saving the value returned by send_import_message.

`result = send_import_message(to_jsonapi)`